oeps/oep-0045/decisions/0001-tutor-as-replacement-for-edx-configuration.rst

```diff
@@ -0,0 +1,494 @@
+Tutor as a replacement for ``edx/configuration``
+================================================
+
+
+Status
+------
+
+Draft
```

Comment:
@arbrandes @nasthagiri As discussed during the contributors meetup yesterday, should we use "Provisional" as a status for this ADR? And maybe indicate a timeline for taking a more definite decision on it?

Reply:
I am supportive of this.